src/account/Domain/UserOrganization.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain
+{
+	public class UserOrganization
+	{
+		public Guid Id { get; set; }
+		public string Name { get; set; }
+		public string Address { get; set; }
+		public string Info { get; set; }
+
+		public ApplicationUser user { get; set; }
```

Comment:
ApplicationUser property name should be Owner. Also, it should have OwnerId with Guid type